src/lobsterpy/cli.py

```diff
@@ -853,6 +1020,107 @@ def run(args):
             orbital_resolved=args.orbitalresolved,
         )
 
+    if args.action in ["plot-bwdf", "plotbwdf"]:
+        if args.cobis:
+            filename = get_file_paths(path_to_lobster_calc=Path(os.getcwd()), requested_files=["icobilist"]).get(
+                "icobilist"
+            )
+            options = {"are_cobis": True, "are_coops": False}
+        elif args.coops:
+            filename = get_file_paths(path_to_lobster_calc=Path(os.getcwd()), requested_files=["icooplist"]).get(
+                "icooplist"
+            )
+            options = {"are_cobis": False, "are_coops": True}
+        else:
+            filename = get_file_paths(path_to_lobster_calc=Path(os.getcwd()), requested_files=["icohplist"]).get(
+                "icohplist"
+            )
+            options = {"are_cobis": False, "are_coops": False}
+
+        structure_filename = get_file_paths(path_to_lobster_calc=Path(os.getcwd()), requested_files=["structure"]).get(
```

Comment:
Why do we actually use os and Path here?

Reply:
Ah both are Not really needed, will clean it up. Just os.getcwd() should also be enough

Reply:
Done

Reply:
> Why not Path.cwd() ?

Done.